tests/multipleBackend/objectPutCopyPart.js

```diff
@@ -300,7 +299,7 @@ function testSuite() {
     itSkipCeph('should return error 403 AccessDenied copying part to a ' +
     'different AWS location without object READ access',
     done => {
-        const errorPutCopyPart = { code: 'AccessDenied', statusCode: 403 };
+        const errorPutCopyPart = errors.AccessDenied;
```

Comment:
We can skip the `const` declaration here and just shove `errors.AccessDenied` in the function call